src/XrdPfc/XrdPfcConfiguration.cc

```diff
@@ -274,6 +274,9 @@ bool Cache::Config(const char *config_filename, const char *parameters)
    const char *theINS = getenv("XRDINSTANCE");
    m_isClient = (theINS != 0 && strncmp("*client ", theINS, 8) == 0);
 
+   // Tell everyone else we are a caching prooxy
```

Comment:
There's a typo here: prooxy

Reply:
Sigh.  I am "blessed" with a MacBook butterfly keyboard so it enjoys inserting double or triple characters.

Fixed the typo, rebased to avoid polluting history for a typo.